src/main/java/com/google/code/or/common/util/MySQLUtils.java

```diff
@@ -63,20 +63,23 @@ public static int toYear(int value) {
 		final int m = (int)(value % 100);
 		final int h = (int)(value / 100);
 		final Calendar c = Calendar.getInstance();
-        c.set(1970, 0, 1, h, m, s);
-        c.set(Calendar.MILLISECOND, 0);
-        return new java.sql.Time(c.getTimeInMillis());
+		c.set(1970, 0, 1, h, m, s);
+		c.set(Calendar.MILLISECOND, 0);
+		return new java.sql.Time(c.getTimeInMillis());
 	}
 
-	public static java.sql.Time toTime2(int value, int fraction, int width) {
+	public static java.sql.Timestamp toTime2(int value, int fraction, int width) {

```

Comment:
better to create a `toTimestamp` method than to change the signature of a public interface